backend/zeno_backend/database/select.py

```diff
@@ -1945,6 +1945,40 @@ def slice_or_tag_table(
         return SQLTable(table=filter_results, columns=columns)
 
 
+def slice_instance_ids(
+    project_uuid: str, filter_sql: sql.Composed | None, id_column: ZenoColumn
+) -> list[str]:
+    """Get all data ids for a slice.
+
+    Args:
+        project_uuid (str): project the slice belongs to.
+        filter_sql (sql.Composed | None): filter specified by the slice.
+        id_column (ZenoColumn): identifier of the instance id column in the project.
+
+    Returns:
+        list[str]: list of instance ids that belong to the slice.
+    """
+    with Database() as db:
+        if db.cur is None:
+            return []
+
+        filter = sql.SQL("")
+        if filter_sql is not None:
+            filter = sql.SQL("WHERE ") + filter_sql
+
+        final_statement = sql.SQL(" ").join(
```

Comment:
Can this just be:

`sql.SQL("SELECT {} FROM {}").format(sql.Identifier(id_column.id), sql.Identifier(uuid)))`